article/app/views/liveblog/liveBlogBlocks.scala.html

```diff
@@ -42,4 +42,7 @@ <h2 class="block-title" itemprop="headline">@title</h2>
         }
         @views.html.fragments.share.blockLevelSharing(s"block-${block.id}", article.sharelinks.elementShares(s"block-${block.id}", None), article.metadata.contentType, amp)
     </div>
+    @block.attributes.membershipPlaceholder.map { membershipPlaceholder =>
```

Comment:
you might want to use `_` rather than `membershipPlaceholder` to make it clear the value is not used